docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -549,15 +549,15 @@ children:
       url: DashDB.html
       output: 'web, pdf'
 
-    - title: 'DB2 connector'
+    - title: 'IBM Db2 (for Linux, Unix, Windows) connector'
       url: DB2-connector.html
       output: 'web, pdf'
 
-    - title: 'DB2 for iSeries connector'
-      url: DB2-iSeries-connector.html
+    - title: 'IBM Db2 for i connector'
+      url: DB2-for-i.html
```

Comment:
Ditto, I think the link is still `DB2-iSeries-connector.html` as we haven't changed the permalink of the page.

Reply:
We'll be removing `loopback-connector-db2iseries` (i.e. DB2 for iSeries) from the list, and adding the `loopback-connector-ibmi` (Db2 for i). See discussion in https://github.com/strongloop/loopback-next/issues/4371. 

The `DB2-for-i.html` will be added in https://github.com/strongloop/loopback.io/pull/943.

Reply:
Sounds good 👍 mark it as resolved